### PR TITLE
Timeout(Fabric function) and Remote Debug for SAIC-239

### PR DESCRIPTION
**Pull request by mirrorcoder**

### CODE REVIEW

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Functional test failure

**Comment by Wasapl** [comment]
Functional test failure

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by Wasapl** [comment]
Build finished.
Unittest success

**Comment by Wasapl** [comment]
Unittest success

**Comment by realsystem** [comment]
Can you, please, add TODO in the code to make sure that we will remove this after fixing the issue? And also make a good commit message and the headline of pull-request.

File: cloudferrylib/utils/drivers/ssh_chunks.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 from cloudferrylib.utils import driver_transporter
 from cloudferrylib.utils import utils
+import sys
```
Comment (realsystem) [inline cloudferrylib/utils/drivers/ssh_chunks.py]: Why we need this lib here?

**Comment by roman-verchikov** [comment]
The patch doesn't seem to be fixing the root cause of the issue. Let's eliminate the root cause instead of providing doubtful workarounds.

**Pull request closed without merging**